src/nunit.analyzers.tests/ConstActualValueUsage/ConstActualValueUsageAnalyzerTests.cs

```diff
@@ -0,0 +1,137 @@
+using Gu.Roslyn.Asserts;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Analyzers.ConstActualValueUsage;
+using NUnit.Framework;
+
+namespace NUnit.Analyzers.Tests.ActualConstValueUsage
+{
+    public class ActualConstValueUsageAnalyzerTests
```

Comment:
I think we should rename the class to follow the file name.
```suggestion
    public class ConstActualValueUsageAnalyzerTests
```